src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -252,35 +220,35 @@ export const canShow = async (
 		};
 	}
 
-	const { brazeSwitch } = window.guardian.config.switches;
-	const apiKey = window.guardian.config.page.brazeApiKey;
-	const isBrazeConfigured = brazeSwitch && apiKey;
-	if (!isBrazeConfigured) {
-		return { result: false };
-	}
+	const dependenciesResult = await checkBrazeDependencies(
+		isSignedIn,
+		idApiUrl,
+	);
 
-	const [brazeUuid, hasGivenConsent] = await Promise.all([
-		asyncBrazeUuid,
-		hasRequiredConsents(),
-	]);
+	if (!dependenciesResult.isSuccessful) {
+		const { failureData, failureField, data } = dependenciesResult;
+		if (SDK_OPTIONS.enableLogging) {
+			// eslint-disable-next-line no-console
+			console.log(
```

Comment:
`console.error` might be better?

Reply:
It's actually not super exceptional to end up here, so an error doesn't feel quite right to me (for example, any user who isn't logged in won't meet all the dependencies, so will end up here). This is mostly useful to a developer working on the code (also in the wild `SDK_OPTIONS.enableLogging` won't be enabled).